CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.6.0] - 2024-07-01
+
+### Refactoring
+
+- [**breaking**] Custom flow into configurable oauth
```

Comment:
I would change both the changelog entry and the commit message here to stay consistent with imperative mood: `refactor!: Rewrite custom flow into configurable oauth` or something

Reply:
I don't mind, but here's a counter argument: the message already contains a verb, so if we replace "Rewrite" with more technical term "Refactor", we get `refactor!: Refactor something something`. So my question is why do we have to duplicate the verb? `fix` and `release` are similar, `feat` however is a noun, but you can still read it as "add feature"

Reply:
So, when using conventional commits, the first part of the commit is not part of the commit description, but rather just a commit type and an optional scope. In that sense, `refactor`, fix` and `release` are actually both verbs and nouns, but in this case it is the noun that is intended. The actual description part of the commit message doesn't start until after the colon, and here the imperative mood should be used.

https://www.conventionalcommits.org/en/v1.0.0/#summary

Reply:
There's nothing about imperative mood, and there's this example:
> fix: array parsing issue when multiple spaces were contained in string.

It's similar to 
> refactor!: custom flow

and 
> refactor!: custom flow into oauth

Reply:
well, imperative mood in the commit message is not part of conventional commits, that's just widely established good practice imo